drivers/mmcsd/sdio.c

```diff
@@ -407,6 +408,25 @@ int sdio_probe(FAR struct sdio_dev_s *dev)
   /* Receive R4 response */
 
   ret = SDIO_RECVR4(dev, SDIO_CMD5, &data);
+
+  if (ret != OK)
+    {
+      goto err;
+    }
+
+  bit = ffs(data);
```

Comment:
Please include comment before this block

Reply:
done